src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,108 @@
+import {useState} from 'react';
+import {IconContext} from 'react-icons';
+import * as FaIcons from 'react-icons/fa';
+import * as MdIcons from 'react-icons/md';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+import {NavigationData} from './NavigationData';
+
+export default function Navigation() {
+  const [showSidebar, setShowSidebar] = useState(false);
+
+  const toggleSidebar = () => setShowSidebar(!showSidebar);
+  return (
+    <>
+      <IconContext.Provider value={{color: '#FFF'}}>
+        <MenuDiv>
+          <MenuLink to="#">
+            <FaIcons.FaBars onClick={toggleSidebar} />
+          </MenuLink>
+        </MenuDiv>
+        <NavMenu isOpen={showSidebar}>
+          <MenuList>
+            <MenuItem>
+              <MenuLinkClose to="#">
+                <SpanClose>
+                  <MdIcons.MdClose onClick={toggleSidebar} />
+                </SpanClose>
+              </MenuLinkClose>
+            </MenuItem>
```

Comment:
Why do you use `MenuLink` and the within that an icon that is clickable? Links and buttons are two very different elements and have different functionality. 

Why do you need to link to nothing here? Can't you use a button instead?
```suggestion
          <MenuButton onClick={toggleSidebar} aria-label="Open menu">
            <FaIcons.FaBars />
          </MenuButton>
        </MenuDiv>
        <NavMenu isOpen={showSidebar}>
          <MenuList>
            <MenuItem>
              <MenuButtonClose onClick={toggleSidebar} aria-label="Close menu">
                <SpanClose>
                  <MdIcons.MdClose />
                </SpanClose>
              </MenuButtonClose>
            </MenuItem>
```
This way it would be a bit more accessible, I think.